lib/FullScreen.js

```diff
@@ -101,6 +104,14 @@ function (_ImageViewer) {
         this.load(imageSrc, hiResImageSrc);
       } // handle window resize
 
+      // escape listener
+      this._events.escapeListener = (e) => {
+        if (e.keyCode === 27 || e.key === 'Escape') {
+          this.hide();
+          e.preventDefault();
+        }
+      }
+      document.addEventListener('keyup', this._events.escapeListener);
```

Comment:
use assignEvent from the util, which returns a method to remove event.
You can do 
```js
this.__events.onEscape = assignEvent(document, 'keyup', this._events.escapeListener);
```
and in hide method you can just call.
```js
this.__events.onEscape();
```